.github/ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,20 @@
 ## Description
-Add a brief and meaningful description.
+<!-- Add a brief and meaningful description. -->
 
 ## Expected Behavior
+<!-- Describe the expected behaviour. -->
 
 ## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
 
-## Steps to Reproduce
-1.
-2.
+## Environment
 
-## Specifications
-  - Browser:
-  - Browser version:
-  - OS: 
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
```

Comment:
Not sure this always applies to the web app repo? Example: for UXD, operating system and browser are the most important, and running locally is done using `yarn`.